packages/fxa-settings/src/components/AppLayout/index.test.tsx

```diff
@@ -7,15 +7,30 @@ import React from 'react';
 import { render } from '@testing-library/react';
 import { MockedCache } from '../../models/_mocks';
 import AppLayout from '.';
+import {createHistory, createMemorySource, LocationProvider} from "@reach/router";
 
-it('renders the app with children', () => {
-  const { getByTestId } = render(
+function renderWithRouter(
```

Comment:
@jodyheavener @LZoog @dannycoates is there a better way to mock providers?

Reply:
idk 🤷

Reply:
`history={history}` can become {...{history}}`, but without looking/poking further I don't know of a better way besides perhaps sticking this function in another file to import so we don't have it defined x3 times?

Reply:
@vbudhram if you pull latest, we do have `models/_mocks` with a `MockedCache` provider there where a helper for this could live. There's also a [mocked AlertBarContext provider](https://github.com/mozilla/fxa/blob/main/packages/fxa-settings/src/lib/AlertBarContext.tsx#L24-L34) in another file that could be moved here to get all our mocked providers in one spot.